python/baseline/tf/classify/model.py

```diff
@@ -403,6 +404,7 @@ def create(cls, embeddings, labels, **kwargs):
                 with tf.variable_scope("output"):
                     model.logits = tf.identity(fully_connected(stacked, nc, activation_fn=None), name="logits")
                     model.best = tf.argmax(model.logits, 1, name="best")
+                    model.probs = tf.nn.softmax(model.logits, name="probs")
```

Comment:
For backwards compat, I would think that `load()` needs to be updated to create this if its not found

Reply:
Fixed. tested by training a model on master and reloading it in this branch